mods/default/README.txt

```diff
@@ -185,6 +187,9 @@ asl97 (WTFPL):
 KevDoy (CC BY-SA 3.0)
   heart.png
 
+Everamzah (CC BY-SA 3.0)
```

Comment:
Please change this line according to this: https://github.com/minetest/minetest_game/pull/1093#issuecomment-219686804